ephios/api/signals.py

```diff
@@ -0,0 +1,10 @@
+from django.dispatch import receiver
+
+from ephios.core.signals import periodic_signal
+
+
+@receiver(periodic_signal, dispatch_uid="ephios.api.signals.clear_expired_tokens")
+def clear_expired_tokens(sender, **kwargs):
+    from oauth2_provider.models import clear_expired
+
+    clear_expired()
```

Comment:
For plugins, we import `signals.py` in the app config, but that's not happening with the `api` app. Please make sure this file is imported.

Reply:
Why are we doing this? Wouldn't it be helpful to retain expired tokens for a few days for debugging/forensics?

Reply:
not yet. It's fine for now I guess. Still needs the signals import though.